suite/tests/api/ir_rv64_addr.templatex

```diff
@@ -0,0 +1,19 @@
+lui    0x2a -> a0
+auipc  <rel> 0x[0-9a-f]+ -> a0
```

Comment:
Can we have a programmatic check of the right PC-relative value: by checking the operand when we decode inside the test?  I know we have the `instr_same` check; but pulling out the address and checking vs a named constant used to construct it feels like a nice extra check.

Reply:
> I know we have the `instr_same` check; but pulling out the address and checking vs a named constant used to construct it feels like a nice extra check.

Maybe we could extract the address part from the decoded result and assert it's equal to the original value. But I really don't think it's meaningful since we already have `instr_same()` checks.

Another possible solution that strace is using[1],

- Print the expected output in the test program
- compare it with the decoded result

It sounds neat and provides a very strong check, but messes the code up and requires a significant change of current test infra.

I'd like to leave it as is (at least for now). Maybe it's appropriate to add a TODO here?

[1]: https://github.com/strace/strace/blob/ba41bc0da4b841d9343b7644a3d8c3bd5e3f2780/tests/riscv_flush_icache.c#L56-L58

Reply:
Sure, ok.